pithos/pithos.py

```diff
@@ -195,7 +196,8 @@ def init_core(self):
         self.player_status = PlayerStatus()
 
         self.stations_dlg = None
-
+        self.buffering = False
+        self.player_paused = False
```

Comment:
I make use of the player_status thing a lot in my fork. I was just going for the bare minimum as far as changes. Baby steps and all for commits right,lol.